<launch>
  <!-- gmsl multiple cameras nodelet -->
  <node pkg="nodelet" type="nodelet" name="camera_nodelet_mrg" args="manager" output="screen"/>
 
  <node pkg="nodelet" type="nodelet" name="camera_nodelet" 
        args="load gmsl_camera/CameraNodelet camera_nodelet_mrg" output="screen">
    <param name="image_width" value="640" />                                                       
    <param name="image_height" value="480" />                                                                
    <param name="image_rate" value="30" />
    <param name="image_buffer" value="5" />                                                               
    <param name="type_ab" value="ar0231-rccb" type="str" />                                              
    <param name="type_cd" value="ar0231-rccb" type="str" />                                                  
    <param name="type_ef" value="ar0231-rccb" type="str" />                                             
    <param name="selector_mask" value="11111111" type="str" />                                               
    <param name="cross_csi_sync" value="0" type="str" />                                                
    <param name="fifo_size" value="3" type="str" />                                                          
    <param name="slave" value="0" type="str" />           
  </node>  

</launch>
